neofetch

```diff
@@ -1375,8 +1375,11 @@ get_title() {
     user=${USER:-$(id -un || printf %s "${HOME/*\/}")}
 
     case $title_fqdn in
-        on) hostname=$(hostname -f) ;;
-        *)  hostname=${HOSTNAME:-$(hostname)} ;;
+        on) hostname="$(uname -n)" ;;
+        *)  hostname="${HOSTNAME:-"$(
+                hostname="$(uname -n)"
+                printf '%s\n' "${hostname%%.*}"
+            )"}" ;;
```

Comment:
You need to keep `hostname -f` as `uname -n` cannot print the FQDN. `uname -n` is fine for non-FQDN output though.